src/crimson/os/seastore/cache.h

```diff
@@ -296,6 +296,31 @@ class Cache {
 	std::move(ret));
     }
 
+    if (!cached->has_actual_data()) {
```

Comment:
oops, this is why ut failed because retiredplaceholder is also a not fully loaded extent. So when another transaction read out this retiredplaceholder in cache, it will be treated as a not fully loaded extent, then to load buffer.